desktop/src/apps/DecisionsApp.test.tsx

```
@@ -58,6 +58,8 @@ describe("DecisionsApp", () => {
       mockFetch({
         "GET /api/decisions?status=pending": { ok: true, body: [singleSelect] },
         "GET /api/decisions?status=answered": { ok: true, body: [] },
+      "GET /api/agents/auth-requests?status=pending": { ok: true, body: { requests: [] } },
```

Biome (2.5.1)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)